src/main/java/org/killbill/billing/overdue/api/OverdueCondition.java

```diff
@@ -35,4 +35,6 @@
     public PaymentResponse [] getResponseForLastFailedPaymentIn();
 
     public ControlTagType getControlTagType();
```

Comment:
👍 (I'm fine breaking the XML since it wasn't implemented)

@matias-aguero-hs Let's not forget to update the release notes ;-)